frontend/coprs_frontend/config/copr.conf

```diff
@@ -207,6 +207,15 @@ HIDE_IMPORT_LOG_AFTER_DAYS = 14
 # failed builds.  Currently only integrated with the Fedora Copr instance.
 #LOG_DETECTIVE_BUTTON = False
 
+# After a given _INACTIVITY_WARNING period (DAYS), when no new build appeared
+# new build appeared in a rolling chroot, we start the EOL policy.  It means
+# that - after the next _INACTIVITY_REMOVAL DAYS - the chroot gets disabled, and
+# builds the corresponding builds removed to save storage.  This is only
+# applicable to MockChroots.rolling = True, and the ... TODO ... command must be
+# configured in cron.
+#ROLLING_CHROOTS_INACTIVITY_WARNING = 180
+#ROLLING_CHROOTS_INACTIVITY_REMOVAL = 180
```

Comment:
@FrostyX please take a look how often you build in your projects

Reply:
For example these projects:

- https://copr.fedorainfracloud.org/coprs/frostyx/mandown (3 years since the last build)
- https://copr.fedorainfracloud.org/coprs/frostyx/cask (3 years since the last build)
- https://copr.fedorainfracloud.org/coprs/frostyx/light/builds/ (recent build but 4 years window between builds at some point)

I don't actively use `cask` and `mandown` but I tried to install them in a Rawhide container and they both worked. And I've been using `light` since 2017 and it always worked.

Reply:
Sooo, do we need an opt-out knob?  For your use-cases, it's hard to imagine an useful default for those options :-)

Reply:
I don't know if there is something we can do. I (and maintainers in a similar situation) will receive a notification email and will probably have to rebuild the package or click an "extend" button once in a while.

Reply:
Ok, let's create the knob when users ask for it.  The effect of this change won't come earlier than after a year.